cloudify_agent/api/pm/base.py

```diff
@@ -274,6 +287,22 @@ def __init__(self, logger=None, **params):
                               backend=self.broker_url)
         self._celery.conf.update(
             CELERY_TASK_RESULT_EXPIRES=defaults.CELERY_TASK_RESULT_EXPIRES)
+        if self._get_ssl_cert_path() != '':
+            self._celery.conf.BROKER_USE_SSL = {
+                'ca_certs': self._get_ssl_cert_path(),
+                'cert_reqs': ssl.CERT_REQUIRED,
+            }
+
+    def _create_celery_conf(self):
+        utils.render_template_to_file(
+            template_path='pm/shared/worker_conf.py.template',
+            file_path=os.path.join(self.virtualenv,
```

Comment:
We should probably have worker_conf.py as a static file within cloudify-plugins-common so that it is accessible to the rest service as well.
We can then generate the configuration file (yaml) for it which is like cloudify_broker_conf.properties and read it from within worker_conf.py. To set and get the location for the generated properties file we can use pkg_resources.